internal/backend/logger/logger.go

```diff
@@ -15,7 +15,7 @@ type Config struct {
 	Zap zap.Config `koanf:"zap"`
 }
 
-var defaultZapLevel = zap.NewAtomicLevelAt(zap.InfoLevel)
+var defaultZapLevel = zap.NewAtomicLevelAt(zap.DebugLevel)
```

Comment:
see WithDebug, looks like we're using it somewhere and it would do nothing now. Maybe we can just pass '--debug' or something instead?

Reply:
After checking (https://github.com/search?q=repo%3Aautokitteh%2Fautokitteh%20WithDebug&type=code), it looks like `WithDebug` isn't actually used anywhere, so I'm removing it.